Week2/homework/maartjes_work.js

```diff
@@ -43,5 +43,70 @@ const tuesday = [
 ];
 
 const tasks = monday.concat(tuesday);
+/*
+function duration(item) {
+  return item.duration;
+}
+function sum(prev, next) {
+  return prev + next;
+}
+console.log(tasks.map(duration).reduce(sum));
+*/
+
+/*
+let hours = 0;
+for (let i = 0; i < tasks.length; i++) {
+  const num = tasks[i].duration;
+  hours += num;
+}
+hours /= 60;
+console.log(hours);
+*/
+
+let hours = 0;
+const duration = (tasks.map(tasks => tasks.duration));
```

Comment:
You need to be more precise in your naming. When we map a `tasks` array the mapping function receives one `task` at a time. Therefore the argument name should be `task` and not `tasks`. The result of `.map()` is an array. Therefore the result should be assigned to a variable `durations` and not `duration`.

In the assignment is was asked to convert the durations from minutes to hours: _Map the tasks to durations in hours._ So you need to divide by 60.

Finally, there is not need for the extra set of parentheses around enclosing the whole expression:

```js
const durations = tasks.map(task => task.duration / 60);
```